pages/app.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import ImageUploading from '../src';
 
-export function App() {
+export default function App() {
```

Comment:
For the sake of intelligent feature. I recommend use `export` not `export default`